ccm_web/server/src/api/api.interfaces.ts

```diff
@@ -10,3 +12,12 @@ export interface Globals {
     roles: string[]
   }
 }
+
+export interface APIErrorData {
+  statusCode: number
+  message: string
+}
+
+export function isAPIErrorData (value: unknown): value is APIErrorData {
+  return hasKey(value, 'statusCode') && hasKey(value, 'message')
```

Comment:
There's some redundancy here because `hasKey` will check twice whether it's not null and an object, but seemed somewhat harmless and better to use the custom defined method than do it another idiosyncratic way. Maybe that's wrong though, IDK